Tests/L0/Common-MSBuildHelpers/_suite.ts

```diff
@@ -43,6 +43,9 @@ describe('Common-MSBuildHelpers Suite', function () {
         it('(Format-MSBuildArguments) adds VS version property', (done) => {
             psr.run(path.join(__dirname, 'Format-MSBuildArguments.AddsVSVersionProperty.ps1'), done);
         })
+        it('(Format-MSBuildArguments) adds user agent property', (done) => {
```

Comment:
Can you fix the description to be consistent with the file name? The file name contains "MsDeploy". Also, it looks like "MSDeploy" would be better casing.